hail/python/hail/fs/google_fs.py

```diff
@@ -10,9 +10,6 @@
 
 class GoogleCloudStorageFS(FS):
     def __init__(self):
-        if 'GOOGLE_APPLICATION_CREDENTIALS' not in os.environ:
```

Comment:
Where did this get moved to? Or is it not necessary anymore because you pass the credentials some other way?

Reply:
It's not necessary.  This was set when we were using this in the notebook service notebooks.  It should be set by the notebook service when that is working again.

The code currently calls gcsfs (but ultimately I want to replace it with your file system), which tries various methods to find credentials.  These defaults are what we want in most cases (including non-notebook regular use).  See the discussion at gcsfs.GCSFileSystem: https://gcsfs.readthedocs.io/en/latest/api.html#gcsfs.core.GCSFileSystem.